### PR TITLE
[re.grammar] Don't use monospace for ordinary text.

### PR DESCRIPTION
**Pull request by Eelis**
image

### CODE REVIEW

File: source/regex.tex [hunk 1]
```diff
@@ -3692,19 +3692,14 @@
...
-
-\S and [^[:space:]]
-
-\w and [_[:alnum:]]
-
-\W and [^_[:alnum:]]
-\end{codeblock}
+\begin{tabular}{@{\quad}l@{and}l}
+  \tcode{\textbackslash d } & \tcode{ [[:digit:]]} \\[3mm]
+  \tcode{\textbackslash D } & \tcode{ [}\tcode{\^}\tcode{[:digit:]]} \\[3mm]
```
Comment (jensmaurer) [inline source/regex.tex:3697]: Do we, in fact, have to make separate `\tcode`s here? The source looks ugly.

File: source/regex.tex [hunk 2]
```diff
@@ -3692,19 +3692,14 @@
...
-\s and [[:space:]]
-
-\S and [^[:space:]]
-
-\w and [_[:alnum:]]
-
-\W and [^_[:alnum:]]
-\end{codeblock}
+\begin{tabular}{@{\quad}l@{and}l}
+  \tcode{\textbackslash d } & \tcode{ [[:digit:]]} \\[3mm]
```
Comment (jensmaurer) [inline source/regex.tex:3696]: The "3mm" extra spacing seems oddly arbitrary.

Would a smaller change be to escape from the "codeblock" environment and just write @and@ to get a prose-font "and"?

Comment (tkoeppe) [reply source/regex.tex:3696]: I'd like to see that option explored, too. Generally, I tried to get the vertical whitespace be the same everwhere as much as possible, so I'd be reluctant to introduce an explicit special size here.

Comment (Eelis) [reply source/regex.tex:3696]: Ah, after trying a bunch of different ways to escape the codeblock environment, I gave up and used a tabular instead, but @ seems to work like a charm, thanks! :) Updating patch.

**Comment by Eelis** [comment]
Amended to use @-escaping instead of resorting to a tabular environment.

**Pull request merged**